core/iwasm/fast-jit/fe/jit_emit_function.c

```diff
@@ -350,16 +350,19 @@ jit_compile_op_call_indirect(JitCompContext *cc, uint32 type_idx,
     arg_regs[4] = NEW_CONST(I32, func_type->param_cell_num);
     arg_regs[5] = argv;
 
-    if (!jit_emit_callnative(cc, jit_call_indirect, native_ret, arg_regs, 6)) {
+    if (jit_get_last_error(cc)
```

Comment:
Not sure why we should add `jit_get_last_error(cc)` here, if not, which error will it cause?

Reply:
```cpp
1. Before calling `jit_emit_callnative`, we call `pack_argv(cc)`.
2. It may set error in function `pack_argv`.
3. If we continue to call call `jit_emit_callnative` after calling `pack_argv(cc)` failed, it will cause `bh_assert` failed
```

Reply:
So had better move check into `pack_argv`:
```C
static JitReg
pack_argv(JitCompContext *cc)
{
   ...
   if (jit_get_last_error(cc))
       return 0;
   return argv;
}
```
And add check when calling pack_argv:
```C
if (!(argv = pack_argv(cc)))
    goto fail;
```

Reply:
Thank you for you sugestion. I have changed it.